src/ui/screens/system.rs

```diff
@@ -93,8 +95,8 @@ impl ActivatableScreen for SystemScreen {
                 display,
                 row_anchor(1),
                 Box::new(|info: &LinkInfo| match info.carrier {
-                    true => format!("Uplink: {}MBit/s", info.speed),
-                    false => "Uplink: Down".to_string(),
+                    true => format!("UL:  {}MBit/s", info.speed),
+                    false => "UL:  Down".to_string(),
```

Comment:
Split this into a separate commit and write about the _why_ in the commit message (the IP address line got too wide with the button legend so it had to move left without breaking alignment of the rows, so "Uplink" had to become shorter).